rov/movement/controller.py

```diff
@@ -16,17 +16,19 @@ def __init__(self, motor_control):
 
         self.__thruster_values = {}
 
-    def update(self, user_input):
+    def update(self, user_thruster_input):
 
         # TODO: Add master Control Handler
 
         # Thrust Mapper
-        thruster_values = self.thrust_mapper.calculate(user_input)
+        thruster_values = self.thrust_mapper.calculate(user_thruster_input["desired_thrust"])
 
         # TODO: Add thrust limiter
 
         self.thrusters.set(thruster_values)
 
+        return self.thrusters.get()
+
     def stop_thrusters(self):
         self.thrusters.stop()
 
```

Comment:
these updates look okay but can you mention somewhere you're making these changes too? Or submit them in another PR and just get it merged before